crates/ruff_cli/src/commands/run.rs

```diff
@@ -75,6 +77,38 @@ pub(crate) fn run(
         pyproject_config,
     );
 
+    // Create a cache per package, if enabled.
+    let package_caches = if cache.into() {
+        let mut caches = HashMap::new();
+        // TODO: try to merge this with the detection of package roots above or
```

Comment:
Nit: we generally use the `TODO(charlie)` style (here, `TODO(thomas)`). The author of the TODO isn't necessarily responsible for resolving it, but they are considered the most-knowledge person, and so we generally preserve that in the TODO itself.